internal/gapicgen/generator/config.go

```diff
@@ -1514,6 +1514,14 @@ var MicrogenGapicConfigs = []*MicrogenConfig{
 		// GA after 2022/07/10
 		ReleaseLevel: "beta",
 	},
+	{
+		InputDirectoryPath:    "google/cloud/videointelligence/v1p3beta1",
+		Pkg:                   "videointelligence",
+		ImportPath:            "cloud.google.com/go/videointelligence/apiv1p3beta1",
+		GRPCServiceConfigPath: "videointelligence_grpc_service_config.json",
+		ApiServiceConfigPath:  "videointelligence_v1p3beta1.yaml",
```

Comment:
Do you want to add the `// GA after ... ` comment here?

Reply:
nope, this is a beta client and will never be considered GA